docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -306,6 +306,73 @@ Here is an example of AWS Lambda Java function managed with Terraform and the ht
+Creating a trace in the calling invocation of the Lambda function can be achieved using these or other methods that result in `traceparent` header added to the outgoing HTTP call:
+* If the caller is a web browser, instrumenting the caller with [Elastic RUM](https://www.elastic.co/guide/en/apm/get-started/current/rum.html) [enabling distributed tracing on the RUM agent](https://www.elastic.co/guide/en/apm/agent/rum-js/5.x/distributed-tracing-guide.html).
+* If the caller is an application, instrumenting this application with an OpenTelemetry or Elastic APM agent ensuring distributed tracing is enabled.
+
+* The configuration of the OpenTelemetry Collector, with the definition of the Elastic Observability endpoint, can be added to the root directory of the Lambda binaries (e.g. defined in `src/main/resources/opentelemetry-collector.yaml`)
++
+[source,yaml]
+----
+# Copy opentelemetry-collector.yaml in the root directory of the lambda function
+# Set an environment variable 'OPENTELEMETRY_COLLECTOR_CONFIG_FILE' to '/var/task/opentelemetry-collector.yaml'
+receivers:
+  otlp:
+    protocols:
+      http:
+      grpc:
+
+exporters:
+  logging:
+    loglevel: debug
+  otlp/elastic:
+    # Elastic APM server https endpoint without the "https://" prefix
+    endpoint: "${ELASTIC_OTLP_ENDPOINT}" <1>
+    headers:
+      # Elastic APM Server secret token
+      Authorization: "Bearer ${ELASTIC_OTLP_TOKEN}" <1>
+
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [logging, otlp/elastic]
+    metrics:
+      receivers: [otlp]
+      exporters: [logging, otlp/elastic]
+----
+<1> Environment-specific configuration parameters can be conveniently passed in as environment variables: `ELASTIC_OTLP_ENDPOINT` and `ELASTIC_OTLP_TOKEN`
+
+* Configure the AWS Lambda Node.js function with:
+** https://docs.aws.amazon.com/lambda/latest/dg/API_Layer.html[Function
+layer]: The latest https://aws-otel.github.io/docs/getting-started/lambda/lambda-js[AWS
+Lambda layer for OpenTelemetry] (e.g. `arn:aws:lambda:eu-west-1:901920570463:layer:aws-otel-nodejs-ver-0-19-0:1`)
+** https://docs.aws.amazon.com/lambda/latest/dg/API_TracingConfig.html[TracingConfig / Mode] set to `PassTrough`
+** https://docs.aws.amazon.com/lambda/latest/dg/API_FunctionConfiguration.html[FunctionConfiguration / Timeout] set to more than 10 seconds to support the cold start of the Lambda JS Runtime
+** Export the environment variables:
+*** `AWS_LAMBDA_EXEC_WRAPPER="/opt/otel-handler"` for wrapping handlers proxied through the API Gateway (see https://aws-otel.github.io/docs/getting-started/lambda/lambda-js#enable-auto-instrumentation-for-your-lambda-function[here])
```

Comment:
```suggestion
** `AWS_LAMBDA_EXEC_WRAPPER="/opt/otel-handler"` for wrapping handlers proxied through the API Gateway. See https://aws-otel.github.io/docs/getting-started/lambda/lambda-js#enable-auto-instrumentation-for-your-lambda-function[enable auto instrumentation for your lambda-function].
```

Reply:
@EamonnTP why do you want to remove the 3rd level indentation of the bullet list?

Reply:
Hi @cyrille-leclerc This is to fall inline with my other suggestions above; dropping the bullets point down a level. There are still a few other comments that need addressed. Thanks.